### DISCUSSION TITLE
Joint global surge model validation efforts

### DISCUSSION THREAD

**Discussion by saeed-moghimi-noaa** [oceanmodeling/searvey#155] [General]
@pmav99 @tomsail @SorooshMani-NOAA @gseroka  @jwesterink @poseidonND 

All, 

Following our recent conversation, I start this discussion for a possible international joint global tide and surge model validation / skill assessment efforts.

Here are data and associated model runs. More information to follow:

https://noaa-nos-stofs2d-pds.s3.amazonaws.com/_archive/msl_2017/   (2 year old)

We will share our recent 2017 model runs soon.

WL observations:
https://drive.google.com/drive/folders/1dmKmZw2g-sKalRi_V6ZDX4ERdVAg2yV9?usp=sharing

**Comment by gseroka** [comment]
@saeed-moghimi-noaa and all

https://noaa-nos-stofs2d-pds.s3.amazonaws.com/index.html#_archive/msl_2017/ is the corrected link. That is actually for our latest version, but only includes the fort.53 and everdried files from ADCIRC (for MSL purposes) (Elevation Harmonic Constituents at All Nodes in the Model Grid (fort.53)) We'll share the full model runs, etc soon...

Cheers!
Greg

**Reply by saeed-moghimi-noaa** [reply]
Thanks a lot @gseroka

**Comment by tomsail** [comment]
@saeed-moghimi-noaa after today's meeting, I wanted to give a follow up on this here. 

WRT STOFS2D outputs:
So far, we found 71 concomitant stations between the STOFS2D stations and IOC_cleanup. 
Details in this notebook:  https://tomsail.github.io/static/IOC_in_STOFS.html

For more context: 
 * IOC_cleanup is a subset from the IOC database which is only 171 stations that contain really good quality data from 2022 until now. https://github.com/seareport/ioc_cleanup
 * I gave a presentation of our dashboard for the skill assessment of our global model, and how we compare **detided** observations with our surge model ouputs. Presentation is available here:  https://github.com/seareport/skill-panel-demo/wiki

Actions: 
 * I will add metrics from STOFS2D outputs (swl points outputs) for the 71 stations > and put them in the dashboard. 
 * We will publish a new version of `ioc_cleanup` with more stations and more time coverage
 * When published, I will update the dashboard and the notebook above
